### PR TITLE
feat: Add mobile dragging functionality

### PR DESCRIPTION
**Pull request by 22JH**
### feature
- Add mobile touch events
- Add blur event at ProseMirror on mobile (to prevent keyboard from popping up)
- Adjust dot size for mobile (from 10px to 16px)
- Adjust dot position for mobile (from -4px to -10px)

### CODE REVIEW

File: lib/imageResize.ts [hunk 1]
```diff
@@ -207,7 +220,7 @@ export const ImageResize = Image.extend({
           const $dot = document.createElement('div');
           $dot.setAttribute(
             'style',
-            `position: absolute; width: 9px; height: 9px; border: 1.5px solid #6C6C6C; border-radius: 50%; ${dotsPosition[index]}`
+            `position: absolute; width: ${isMobile ? 16 : 10}px; height: ${isMobile ? 16 : 10}px; border: 1.5px solid #6C6C6C; border-radius: 50%; ${dotsPosition[index]}`,
```
Comment (bae-sh) [inline lib/imageResize.ts:223]: I think it would be best to continue using 9px on the web env.

**Review by bae-sh — COMMENTED** [review]
Thanks for your PR! The mobile feature looks fantastic. One small thing—could you fetch the latest changes and apply the Prettier configuration to match printWidth: 100?Appreciate your help!

File: lib/imageResize.ts (hunk 1 above)
Comment (22JH) [reply lib/imageResize.ts]: 😄

Comment (bae-sh) [reply lib/imageResize.ts]: Hi, @22JH Thanks for the update! Just to clarify, I just wanted to replace 10px with 9px for the web version.

Could you also adjust the mobile width and height to 16px again? And please double-check dotsPosition as well.

ex)
```js
 `position: absolute; width: ${isMobile ? 16 : 9}px; height: ${isMobile ? 16 : 9}px; border: 1.5px solid #6C6C6C; border-radius: 50%; ${dotsPosition[index]}`,
```

Comment (22JH) [reply lib/imageResize.ts]: I must have misread this lol
I've adjusted it to 10px for web (min 768px) and 16px for mobile.

**Pull request merged**